doc/tutorials/launch_uos.sh

```diff
@@ -127,32 +127,18 @@ if [ "$setup_mem" != "" ];then
 fi
 
 boot_dev_flag=",b"
```

Comment:
Isn't that a flag that's only for vSBL too?

Reply:
This script mentions a `-V` option to use vSBL. Shouldn't that be removed?

Reply:
Thanks, I have check and remove it.